### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by Tasksqwx** [community/community#121189] [Copilot Conversations] [answered]
### Select Topic Area

Show & Tell

### Body

unhandled status from server: 404 404 page not found

**Comment by Akash1134** [comment, marked as answer]
Hi @Tasksqwx ,

Please let us know the which IDE are you using and what is the Copilot/Copilot Chat version in use.

Also, for instance if you are using Visual Studio, please try upgrading to the latest VisualCode [version](https://code.visualstudio.com/updates/v1_88) along with the latest Copilot/Copilot Chat plugin version, and checking whether this issue still occurs.

Thanks!

**Reply by Tasksqwx** [reply]
<img width="396" alt="image" src="https://github.com/community/community/assets/57429151/0d066cd9-8f7e-45b1-ba7f-e90dbd627783">
in vscode

**Reply by davecheney** [reply]
@Tasksqwx upgrade to 0.14 (or later) of the chat extension
